app/src/main/java/com/github/gotify/messages/MessagesActivity.java

```diff
@@ -120,7 +120,7 @@ public void onReceive(Context context, Intent intent) {
     private boolean isLoadMore = false;
     private Integer selectAppIdOnDrawerClose = null;
 
-    int PICASSO_CACHE_SIZE = 50 * 1024 * 1024; // 50 MB
+    public static int PICASSO_CACHE_SIZE = 50 * 1024 * 1024; // 50 MB
```

Comment:
This constant can be moved to Utils as well.

Reply:
See the other comment regarding chache/picasso handler